Directory.Build.props

```diff
@@ -253,11 +250,13 @@
     <OutputRID Condition="'$(OutputRID)' == ''">$(_outputOS)-$(TargetArchitecture)</OutputRID>
   </PropertyGroup>
 
-  <PropertyGroup Label="CalculateTargetOSName" Condition="'$(SkipInferTargetOSName)' != 'true'">
+  <PropertyGroup Label="CalculateTargetOSName">
```

Comment:
Nit: Would be good to move this up in a follow-up and add TargetsMobile into the group as well.